app/services/flow/merits_loop.rb

```diff
@@ -40,6 +40,7 @@ def convert_task_to_flow_name(task)
         prohibited_steps: :prohibited_steps,
         laspo: :in_scope_of_laspos,
         nature_of_urgency: :nature_of_urgencies,
+        reason_for_new_application: :vary_order,
```

Comment:
It might just be me...

... but I think I prefer `reason_for_new_application` 😄

For me, it conveys more context and meaning than `vary_order` does, but that might be because I have limited domain knowledge.

Having said that, I would advocate for something like `NewApplicationReason` when it comes to model naming.

If the "order" aspect is an important detail, then I think `OrderVariation` works too.

I'd be interested to hear what you (and others) think!

Reply:
this line should have ben removed as i have changed LFA to use vary_order too, for consistency